modules/end-user-guide/pages/optional-parameters-for-the-urls-for-starting-a-new-workspace.adoc

```diff
@@ -0,0 +1,111 @@
+:navtitle: Optional parameters for the URLs for starting a new workspace
+// :page-aliases:
+
+[id="parameters-for-the-urls-for-starting-a-new-workspace_{context}"]
+= Optional parameters for the URLs for starting a new workspace
+
+For the URLs for starting a new workspace, the following optional parameters provide an extra layer of basic workspace configuration in addition to the devfile.
+//here link to a section about the devfile. max-cx
+
+== URL parameter concatenation
+
+The URL for starting a new workspace supports concatenation of multiple URL parameters by using `&` with the following URL syntax:
+[source,subs="+quotes,+attributes"]
+----
+{prod-url}#__<git_repository_url>__?__<url_parameter_1>__&__<url_parameter_2>__&__<url_parameter_3>__
+----
+
+== URL parameter for the workspace IDE
+
+Che-Theia is currently the default integrated development environment (IDE) that a workspace loads with if no IDE is specified as a URL parameter as part of the URL for starting a new workspace.
+
+The URL parameter for specifying another supported IDE is `che-editor=__<editor_key>__`.
+
+The URL for starting a new workspace has the following syntax with the `che-editor=__<editor_key>__` parameter:
+
+[source,subs="+quotes,+attributes"]
+----
+{prod-url}#__<git_repository_url>__?che-editor=__<editor_key>__
+----
+
+.`__<editor_key>__` values for supported IDEs
+|===
+| IDE | `__<editor_key>__` value | Note
+
+ifeval::["{project-context}" == "che"]
+| link:https://github.com/eclipse-che/che-theia[Che-Theia]
+| `eclipse/che-theia/latest`
+| This is the default IDE with link:https://github.com/che-incubator/chectl/[chectl stable]. This IDE loads in a new workspace without entering this URL parameter.
+
+| link:https://github.com/eclipse-che/che-theia[Che-Theia]
+| `eclipse/che-theia/next`
+| This is the default IDE with link:https://github.com/che-incubator/chectl/[chectl next]. This IDE loads in a new workspace without entering this URL parameter.
+endif::[]
+
+ifeval::["{project-context}" == "crw"]
+| link:https://github.com/eclipse-che/che-theia[Che-Theia]
+|`eclipse/che-theia/latest`
+| This is the default IDE: it loads in a new workspace without this URL parameter.
+endif::[]
+
+| link:https://coder.com/docs/code-server/[*code-server*]
```

Comment:
the correct link is https://github.com/che-incubator/che-code

Reply:
:) Thank you! Replaced.